.github/blunderbuss.yml

```diff
@@ -126,6 +126,10 @@ assign_issues_by:
   - 'api: monitoring'
   to: 
   - GoogleCloudPlatform/dee-observability
+- labels:
+  - 'api: kms'
```

Comment:
Please use `cloudkms` instead. Python-kms repo used `cloudkms` by the bot and not `kms`.

Reply:
Done. I left kms and added cloudkms as both show up on the labels list in GitHub.